packages/rpc/src/endpoint.ts

```diff
@@ -247,7 +263,12 @@ export function createEndpoint<T>(
         const resolver = callIdsToResolver.get(callId);
 
         if (resolver == null) {
-          throw new MissingResolverError({callId, error, result});
+          throw new MissingResolverError({
+            callId,
+            error,
+            result,
+            groupingHash: 'MissingResolverError::FunctionResult',
```

Comment:
I was chatting a bit about this with someone in relation Checkout's own error reporting: IMO, trying to distinguish two errors with a "grouping hash" or some other identifier passed in is a very weird pattern. If you need to differentiate between two errors, it should be two different error classes instead.

Reply:
As we don't need to differentiate between two errors, I've updated MissingResolverError to always have the same groupingHash.

I'm happy to update if you think we should differentiate them.